.tours/python/http/.tours/01creating-a-function-app.tour

```diff
@@ -0,0 +1,201 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Python HTTP - 01. Creating a Function App",
+  "steps": [
+    {
+      "file": "src/python/AzureFunctions.Http/01CreatingAFunctionApp/HelloWorldHttpTrigger/__init__.py",
```

Comment:
Please make sure you test everything when you opened the Python HTTP lesson workspace.  The tour files are supposed to be used from within a Workspace, and not from the root folder itself.